ProjectManagmentSystem/package.json

```diff
@@ -10,8 +10,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
     "@fontsource/inter": "^5.0.18",
     "@fortawesome/fontawesome-free": "^6.5.2",
+    "@mui/material": "^5.15.17",
```

Comment:
We will be using MUI in the next project, so we should stick to bootstrap in this project. We also need to uninstall any related dependencies related to it like emotion.